### DISCUSSION TITLE
PAT saving in RStudio Server

### DISCUSSION THREAD

**Discussion by sr320** [RobertsLab/resources#1427] [Q&A]
How do you keep from pasting in PAT every push in RStudio Server?

**Comment by kubu4** [comment]
What is PAT?

**Reply by sr320** [reply]
sorry Personal Access Token

**Reply by kubu4** [reply]
Gotcha!

Try these steps (Starting at Step 3): 

https://gist.github.com/Z3tt/3dab3535007acf108391649766409421

**Comment by kubu4** [comment]
Alternatively, use the Terminal tab in Rstudio and use `git` from the command line.
